bluebottle/tasks/models.py

```diff
@@ -101,6 +105,13 @@ def owner(self):
     def parent(self):
         return self.project
 
+    @property
+    def place(self):
+        try:
+            return self.places.get()
```

Comment:
You could now have multiple places, right?
I think you should also catch `Place.MultipleObjectsReturned` or just do `return self.places.first()` (which is less explicit of course)

Reply:
Ah ne'er mind you limit it in admin too...